integrations/bitdefender/nango.yaml

```diff
@@ -0,0 +1,111 @@
+integrations:
+    bitdefender:
+        syncs:
+            companies:
+                version: 1.0.0
```

Comment:
version is not needed here as it defaults to 0.0.1, the version should only be used once the property exists and a change is made